define/types.go

```diff
@@ -54,6 +54,9 @@ const (
 	SNP TeeType = "snp"
 )
 
+// DefaultRlimitValue is the value set by default for nofile and nproc
+const RLimitDefaultValue = uint64(1048576)
```

Comment:
Where do these values come from?  Are they always the default everywhere?

Reply:
This is the default from Podman, no idea where it came from.

Reply:
@giuseppe  added it

Reply:
Might have been a value from Docker?
https://bbs.archlinux.org/viewtopic.php?id=285058

Reply:
As of 7 years ago was a kernel max.
https://serverfault.com/questions/819119/xtrabackup-too-many-open-files-more-than-1048576

Might want to use something even higher.

cat /proc/sys/kernel/pid_max 
4194304

Reply:
Under Moby I see.
./libnetwork/docs/vagrant-systemd/docker.service:LimitNOFILE=1048576
./libnetwork/docs/vagrant-systemd/docker.service:LimitNPROC=1048576

Reply:
The question is do we continue to roll with it, or should we increase it?

Reply:
I'd vote to keep Podman and Buildah in lockstep.  I don't know if there's a place to tuck the default values so both could reference them from the same place, that would be nice.  I think 1048576 for the kernel max is good for now.

Reply:
Can we move this constant to c/common?  No need to block this PR on it but we should use every occasion to avoid future code divergence; this PR smells like a good example :)